app/src/main/java/com/kickstarter/viewmodels/ActivityFeedViewModel.java

```diff
@@ -115,15 +114,15 @@ public ViewModel(final @NonNull Environment environment) {
           this.currentUser.isLoggedIn(),
           Pair::create
         )
-        .filter(resumedAndLoggedIn ->
-          resumedAndLoggedIn.first == ActivityEvent.RESUME && resumedAndLoggedIn.second
-        )
+        .map(resumedAndLoggedIn -> resumedAndLoggedIn.second)
+        .filter(loggedIn -> loggedIn)
```

Comment:
we've got a [`isTrue`](https://github.com/kickstarter/android-oss/blob/master/app/src/main/java/com/kickstarter/libs/utils/BooleanUtils.java#L18) utility for this kind of thing. may seem kind of silly to have, but it allows you to write this very declaratively as `.filter(isTrue)`